scripts/jenkins/bench.sh

```diff
@@ -1,11 +1,34 @@
 #!/usr/bin/env bash
-set -euxo pipefail
+set -exo pipefail
 
+## Buildkite specific configuration
+if [ "$CI" == "true" ] ; then
+	export GO111MODULE=on
+
+	# If HOME is not set then use the current directory
+	# that's normally happening when running in the CI
+	# owned by Elastic.
+	if [ -z "$HOME" ] ; then
+		HOME=$(realpath ~)
+		export HOME
+	fi
+
+	# Make sure gomod can be deleted automatically as part of the CI
+	clean_up () {
+	  ARG=$?
+	  # see https://github.com/golang/go/issues/31481#issuecomment-485008558
+	  chmod u+w -R $HOME/go/pkg
```

Comment:
I used `GOPATH` but

```
+ ARG=127
--
  | ./scripts/jenkins/bench.sh: line 19: GOPATH: unbound variable
```

Hence since this section runs within the CI context only, we can use HOME in this particular case